src/views/blocks/tickets/extra-available.php

```diff
@@ -25,15 +26,7 @@
 	return;
 }
 
-/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
-$tickets_handler = tribe( 'tickets.handler' );
-
-$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
-
-if ( -1 === $available ) {
-	return;
-}
-
+$available = $ticket->available();
 $post_id   = $this->get( 'post_id' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post_id

<sub>hash: 22733ecf04f7dd562db90db1dfd054f9</sub>